### ISSUE TITLE
idea: docker image for builder?

### ISSUE THREAD

**Issue by dominictarr** [Widdershin/positron#4]
so that you don't have to go through the hassel of setting up all the android stuff it might be easier to run it in a docker on a vps, which would also make it way more reproducable?

**Comment by Widdershin** [comment]
Good idea. I haven't worked with docker before but I would like to support this use case.
